.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+os:
+  - linux
+# ** patchelf isn't supported here, _yet_ **
+# addons:
+#   apt:
+#     sources:
+#       - ppa:utlemming/rax-playground
+#     packages:
+#       - build-essential
+#       - ninja-build
+#       - patchelf
+#       - libboost-dev
+#       - libboost-program-options-dev
+#       - libogg-dev
```

Comment:
Because we need patchelf, we can't use their container infrastructure yet, which supports installing packages via the apt addon. I was playing around with it, this could be deleted. Though once patchelf is made more widely available, we'll want to get rid of before_install: and replace it with the addons: above.